Parent/Parent/Settings/SettingsViewController.swift

```diff
@@ -139,6 +149,8 @@ class SettingsViewController: UIViewController {
                     self.env.router.show(alert, from: self, options: .modal())
                 } else {
                     self.observeesViewController?.refresh()
+                    let observees = AppEnvironment.shared.subscribe(GetObservedStudents(observerID: AppEnvironment.shared.currentSession?.userID ??  "")) { }
+                    observees.exhaust()
```

Comment:
I would expect this to trigger the dashboard to update with the new student.. Could it be because the Store is going out of scope so the use case is as well? Maybe we should just send a Notification with NotificationCenter and have the Dashboard call `students.exhaust()` again or something?